imports/plugins/included/payments-stripe-connect/client/settings/settings.js

```diff
@@ -0,0 +1,45 @@
+import { Template } from "meteor/templating";
+import { Reaction, i18next } from "/client/api";
+import { Packages } from "/lib/collections";
+import { StripeConnectPackageConfig } from "../../lib/collections/schemas";
+
+import "./settings.html";
+
+Template.stripeConnectSettings.helpers({
+  StripeConnectPackageConfig() {
+    return StripeConnectPackageConfig;
+  },
+  packageData() {
+    return Packages.findOne({
+      name: "reaction-stripe-connect",
+      shopId: Reaction.getShopId()
```

Comment:
the stripe connect settings are in the dashboard for owner. are sellers even allowed to edit payment gateway stuff? if so we can change it, just thought this package would only ever be edited by parent shop owner.